packages/@aws-cdk/aws-iam/package.json

```diff
@@ -57,7 +57,8 @@
     "env": {
       "AWSLINT_BASE_CONSTRUCT": "true"
     },
-    "jest": true
+    "jest": true,
+    "rosetta": "strict"
```

Comment:
Wait, does this mean we do a rosetta-compile per package? Isn't that slow?

Reply:
It does, and I don't know. I'll actually be measuring the actual cost there so we can decide based on data. For the sake of this draft at least, I did not want to bother with that problem.

Reply:
Okay it's __VERY__ slow (on `@aws-cdk/core` this amounts to 57.6 seconds, where `jsii` execution alone is a wee 7.9 seconds).

We'll probably have to chat about how we want the selectivity to work... Because tradeoffs are involved... But that'll definitely be baked in before this is allowed to ship.